e2e/test-plugins/grafana-extensionstest-app/webpack.config.ts

```diff
@@ -1,7 +1,8 @@
 import CopyWebpackPlugin from 'copy-webpack-plugin';
-import grafanaConfig from '@grafana/plugin-configs/webpack.config';
+// @ts-ignore - node needs the extension to strip types successfully
```

Comment:
hmm. i think there is a magic tsconfig incantation to fix this... let me see if i can dig it up

Reply:
```json
{
  "compilerOptions": {
    "allowImportingTsExtensions": true
  }
}
```

Not sure if we want that in the existing tsconfig, or in one specific for the webpack config.... Not sure if there's an easy way to get vscode to use a specific tsconfig for just one file that's not in its own folder?

Reply:
I think we need TS 5.8 for that, no?

Reply:
Actually - `allowImportingTsExtensions` [came from Typescript 5.0](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-5-0.html#allowimportingtsextensions).

I gave it a try, and it works. Except it also requires `noEmit` to be specified. I'm not sure of the implications of that for grafana-extensionstest-app...